unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExploreHUD/ExploreHUDController.cs

```diff
@@ -5,7 +5,7 @@
 
 public class ExploreHUDController : IHUD
 {
-    internal ExploreHUDView view;
+    public ExploreHUDView view;
```

Comment:
If we need this `public` can we at least limit the access to set `view`
```suggestion
    public ExploreHUDView view { get; private set; }
```

Reply:
Ok, we can limit the access through a property. Done! Thanks!